---
title: "gke/ERR/2023_006"
linkTitle: "ERR/2023_006"
weight: 1
type: docs
description: >
  GKE Gateway controller reporting misconfigured annnotations in Gateway resource
---

**Product**: [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine)\
**Rule class**: ERR - Something that is very likely to be wrong

### Description

Gateway controller creates loadbalancing resources based on annotations
specified in gateway resouces.

It expects the user to use correct set of supported annotations name and values.

It might be case that annotation name has a typo or mistake.

You can use the following filter to find matching log entries with gateway resource
containing misconfigured annotations:

```
jsonPayload.MESSAGE =~"Error GWCER101: Annotation"
resource.type="k8s_cluster"
```

### Remediation

- Fix the annotaions in gateway resources obtained from above query

### Further information
